src/java/org/apache/cassandra/auth/IRoleManager.java

```diff
@@ -152,6 +152,15 @@ void revokeRole(AuthenticatedUser performer, RoleResource role, RoleResource rev
                                        .collect(Collectors.toSet());
      }
 
+    /**
+     * Return the set of roles that are member of the given role.
```

Comment:
Is it meant to be recursive?

Reply:
Not that I am aware; are you suggesting the description be more clear about that? 👍🏻 
Let me know if you meant something else.